src/main/scala/gitbucket/core/servlet/GitRepositoryServlet.scala

```diff
@@ -523,6 +512,51 @@ class WikiCommitHook(owner: String, repository: String, pusher: String, baseUrl:
     }
   }
 
+  private[this] def buildActionInfo(
+    commit: JGitUtil.CommitInfo,
+    action: String,
+    fileName: String
+  ): Option[WikiActivityActionInfo] = {
+    action match {
+      case "created" =>
+        Some(
+          CreateWikiActivityActionInfo(
+            owner,
+            repository,
+            commit.committerName,
+            fileName.dropRight(".md".length)
+          )
+        )
+      case "edited" =>
+        Some(
+          EditWikiActivityActionInfo(
+            owner,
+            repository,
+            commit.committerName,
+            fileName.dropRight(".md".length),
+            commit.id
+          )
+        )
+      case "deleted" =>
+        Some(
+          DeleteWikiActivityActionInfo(
+            owner,
+            repository,
+            commit.committerName,
+            fileName.dropRight(".md".length),
+            commit.id
+          )
+        )
+      case _ => None
+    }
+  }
+
+  private[this] def mapToAction(changeType: ChangeType): String = changeType match {
```

Comment:
Just wanted to make sure that these are all the actions we currently want to support.